inference-engine/src/mkldnn_plugin/nodes/mkldnn_rnn.cpp

```diff
@@ -68,6 +71,18 @@ size_t statesCount(algorithm alg) {
     }
 }
 
+bool haveCellState(algorithm alg) {
+    return alg == algorithm::vanilla_lstm;
+}
+
+const std::map<InferenceEngine::Precision, InferenceEngine::Precision> MKLDNNRNN::weightsByLayerPrec {
```

Comment:
shouldn't it be static?

Reply:
static is allowed only in scope of class declaration in this case

Reply:
In this case we can have it static only in scope of class declaration.